src/core/AutoRest.Core/Properties/Resources.resx

```diff
@@ -273,6 +308,9 @@
   <data name="PathCannotBeNullOrEmpty" xml:space="preserve">
     <value>path cannot be null or an empty string or a string with white spaces while getting the parent directory</value>
   </data>
+  <data name="GuidUsageNotValid" xml:space="preserve">
+    <value>Guid used at the #/Definitions/{1}/.../{0}. Usage of Guid is not recommanded</value>
```

Comment:
Lets add -"If GUIDs are absolutely required in your service, please get sign off from the Azure API review board."